### PR TITLE
fix: nft and general fixes

### PR DESCRIPTION
**Pull request by alessey**
**What changed? Why?**
* Removed Base NFT, split base styles into Mint/View cards as they have different styles
* Renamed NFTView->NFTCard, NFTMint->NFTMintCard
* small type name fixes
* added NFTCollectionTitle
* View links to opensea
* added paymaster
* fix active styles
* 
**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Review by cpcramer — APPROVED** [review]

File: src/nft/components/NFTCard.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const handleOnClick = useCallback(() => {
+    const network = chain?.name.toLowerCase() ?? 'base';
+    const openSeaUrl = `https://opensea.io/assets/${network}/${contractAddress}/${tokenId}`;
+    window.open(openSeaUrl, '_blank', 'noopener,noreferrer');
+  }, [chain, contractAddress, tokenId]);
+
+  // prevents SSR hydration issue
+  if (!isMounted) {
+    return null;
+  }
```
Comment (abcrane123) [inline src/nft/components/NFTCard.tsx:35]: sorry if i missed this in the other PR but did we decide that we needed this `isMounted` return in both `NFTCard` and `NFTMintCard` ?

File: src/nft/components/NFTCard.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  onSuccess,
+}: NFTCardReact) {
+  const componentTheme = useTheme();
+
+  const isMounted = useIsMounted();
+
+  const { chain } = useAccount();
+
+  const handleOnClick = useCallback(() => {
+    const network = chain?.name.toLowerCase() ?? 'base';
```
Comment (abcrane123) [inline src/nft/components/NFTCard.tsx:27]: is there somewhere else we can find what `chain` they've configured ? it seems like they would've had to have configured it somewhere right?

File: src/nft/components/NFTCard.tsx (hunk 1 above)
Comment (alessey) [reply src/nft/components/NFTCard.tsx:35]: Originally I had the same styling in both so I could extract the divs and these hooks to a common component, but, that was incorrect, both these cards are pretty different so I need to handle them separately.

File: src/nft/components/NFTCard.tsx (hunk 2 above)
Comment (abcrane123) [reply src/nft/components/NFTCard.tsx:27]: ahh i see makes sense

**Review by abcrane123 — APPROVED** [review]

**Pull request merged**